### DISCUSSION TITLE
About Wireframing issue.

### DISCUSSION THREAD

**Discussion by maanpatel2492** [Swarnimashukla/Automatic-attendance-management-system#52] [Q&A]
Hey,
I have done rough work of wireframing the Attendance management system . I think you should verify the rough design once so that I can proceed further on that.
Where should I saw you the design?

**Comment by Swarnimashukla** [comment]
Hey  @maanpatel2492, I have created a discussion page for this process you can go ahead and share your designs to get feedback before final submission.

**Comment by maanpatel2492** [comment]
Hey, I have made rough wireframe 
[Rough Wireframing.pdf](https://github.com/Swarnimashukla/Automatic-attendance-management-system/files/6146277/Rough.Wireframing.pdf)
Please mention for the improvement!!

**Reply by Swarnimashukla** [reply]
You can go ahead with the pr @maanpatel2492  do follow the guidelines mentioned on discord channel, make pr in resource branch with 1 commit only

**Reply by maanpatel2492** [reply]
I will make pr whithin 2 days.
